Tests/LocalLLMExecutionDemo/LocalLLMExecutionDemo/LocalLLMChatView.swift

```diff
@@ -0,0 +1,56 @@
+//
+// This source file is part of the SpeziML open-source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziLocalLLM
+import SpeziOpenAI
+import SwiftUI
+
+/// Presents a chat view that enables user's to interact with the local LLM.
+struct LocalLLMChatView: View {
```

Comment:
Might make sense to move this into the main target and maybe make it configurable? Maybe in the same target as the download UI?